css/css-gaps/flex/META.yml

```diff
@@ -0,0 +1,6 @@
+links:
+- product: firefox
+  results:
+  - test: flex-gap-decorations-034.html
```

Comment:
Renamed to css/css-gaps/flex/fragmentation/flex-gap-decorations-fragmentation-007.html in https://github.com/web-platform-tests/wpt/commit/6e4bfb4d7c62dd38e8514b83fae84da2a172e791